egs/ami/s5/local/tfrnnlm/RESULTS

```diff
@@ -0,0 +1,25 @@
+Here are some numbers comparing rescoring performances of different RNNLM toolkits
+
+Un-rescored baseline:
+%WER 24.2 | 13098 94476 | 79.2 12.0 8.8 3.4 24.2 57.4 | -0.157 | exp/ihm/nnet3/tdnn_sp/decode_dev/ascore_12/dev_hires.ctm.filt.sys
+%WER 25.4 | 12643 89973 | 77.7 14.0 8.3 3.1 25.4 56.5 | -0.144 | exp/ihm/nnet3/tdnn_sp/decode_eval/ascore_11/eval_hires.ctm.filt.sys
+
+CUED-RNNLM with CE objf:
+%WER 23.4 | 13098 94468 | 79.6 11.6 8.8 3.1 23.4 56.6 | -0.189 | exp/ihm/nnet3/tdnn_sp/decode_dev.rnnlm.ce.cued.lat.4gram/ascore_12/dev_hires.ctm.filt.sys
+%WER 24.5 | 12643 89977 | 78.4 13.4 8.2 2.9 24.5 55.6 | -0.164 | exp/ihm/nnet3/tdnn_sp/decode_eval.rnnlm.ce.cued.lat.4gram/ascore_11/eval_hires.ctm.filt.sys
+
+Mikolov-RNNLM:
+%WER 23.9 | 13098 94471 | 79.4 12.0 8.6 3.3 23.9 57.1 | -0.188 | exp/ihm/nnet3/tdnn_sp/decode_dev.rnnlm.mik.lat.4gram/ascore_12/dev.ctm.filt.sys
+%WER 25.0 | 12643 89986 | 78.1 13.8 8.1 3.1 25.0 55.9 | -0.166 | exp/ihm/nnet3/tdnn_sp/decode_eval.rnnlm.mik.lat.4gram/ascore_11/eval.ctm.filt.sys
+
+Tensorflow-RNNLM:
+1-hidden layer vanilla-rnnlm (local/tfrnnlm/run-vanilla-rnnlm.sh):
+%WER 23.4 | 13098 94468 | 79.8 11.7 8.5 3.2 23.4 56.9 | -0.231 | exp/ihm/nnet3/tdnn_sp/decode_dev.vanilla.tfrnnlm.lat.4gram.0.5/ascore_11/dev_hires.ctm.filt.sys
+%WER 24.5 | 12643 89964 | 78.3 13.1 8.7 2.7 24.5 55.6 | -0.135 | exp/ihm/nnet3/tdnn_sp/decode_eval.vanilla.tfrnnlm.lat.4gram.0.5/ascore_12/eval_hires.ctm.filt.sys
+
+LSTM-RNNLM (local/tfrnnlm/run-lstm-fast.sh):
+%WER 23.4 | 13098 94466 | 79.7 11.6 8.7 3.1 23.4 56.8 | -0.213 | exp/ihm/nnet3/tdnn_sp/decode_dev.unk.fast.tfrnnlm.lat.3gram.0.5/ascore_11/dev_hires.ctm.filt.sys
+%WER 24.6 | 12643 89965 | 78.2 13.3 8.6 2.8 24.6 55.6 | -0.148 | exp/ihm/nnet3/tdnn_sp/decode_eval.unk.fast.tfrnnlm.lat.3gram.0.5/ascore_11/eval_hires.ctm.filt.sys
+
+%WER 23.1 | 13098 94461 | 79.9 11.5 8.6 3.0 23.1 56.5 | -0.212 | exp/ihm/nnet3/tdnn_sp/decode_dev.unk.fast.tfrnnlm.lat.4gram.0.5/ascore_11/dev_hires.ctm.filt.sys
```

Comment:
One is 3-gram and the other is 4-gram. I included the 3-gram results because 3-gram LSTM seems to do as well as 4-gram Vanilla RNNLM.